R/esqlabs-plot-export-configuration.R

```diff
@@ -0,0 +1,68 @@
+#' @title ExportConfiguration
+#' @description R6 class defining properties for saving a `ggplot` object
+#' @field name character defining the name of the file to be saved (without extension)
+#' @field path Path of the directory to save plot to: path and filename are
+#' combined to create the fully qualified file name. Defaults to the working directory.
+#' @field format character defining the format of the file to be saved
+#' @field width numeric values defining the width in `units` of the plot dimensions after saving
+#' @field height numeric values defining the height in `units` of the plot dimensions after saving.
+#' Only used if `heightPerRow` is `NULL`.
+#' @field units character defining the unit of the saving dimension
+#' @field dpi (dots per inch) numeric value defining plot resolution
+#' @export
+#' @import ggplot2
+#' @family PlotConfiguration classes
+ExportConfiguration <- R6::R6Class(
```

Comment:
I don't know if there is a way to inherit documentation of R6 classes?

I had to extend the class to dynamically calculate the height of the output figures depending on the number of rows of the multi-panel figure.

Reply:
In this case, I don't think so because it comes from another package.